models/org_team.go

```diff
@@ -87,6 +88,7 @@ func (t *Team) IsMember(userID int64) bool {
 }
 
 func (t *Team) getRepositories(e Engine) error {
+	t.Repos = nil
```

Comment:
This does not seem right

Reply:
I need to reset the value otherwise if I call the function twice the slice contains all repos twice.
I could have aborted the loading if the slice was already loaded like here: https://github.com/go-gitea/gitea/blob/7cca1253f7c7bf90582a71c26582d0369b8109ce/models/org_team.go#L54
But I want to be sure the repos are up to date

Reply:
```go
if t.Repos != nil {
    return nil
}
```

Reply:
If we do like you suggest, we must keep in mind that it will keep using the first loaded, maybe outdated, repos in the following calls of `getRepositories`.
Anyway, this should be fine in our use cases today.